docs/users/extend/in-container-configuration.md

```diff
@@ -1,11 +1,12 @@
 ## In-Container home directory configuration
 
-There are a number of reasons that your web container home directory may need custom configuration. You may want your normal git configuration in there, or a composer auth.json, etc.
+Custom shell config (bash or your preferred shell), your usual git configuration, a composer auth.json and more can be achieved within your containers.  Place all your dotfiles in your `~/.ddev/homeadditions` directory and DDev will use these in all your projects' Docker containers.
```

Comment:
The only thing that's wrong with this is that you're just thinking about shell configuration (.profile, .zshrc, stuff like that). But there are many other things that go into a home directory. Of course, an auth.json has nothing to do with shell configuration, so that's a great example.

Reply:
Would it make sense to break out the "In-container Configuration" into two/multiple pages then?  1) Shell configuration, and 2) Other stuff like auth.json.

Or perhaps to just expand more on what that other stuff is in that section?  I'm not at all aware of what else could be done.

Reply:
It's always a tradeoff - do you try to teach people how to use the shell and every single way you can use it? It's a hopeless task I think.

This feature is actually for advanced users, so I'm not very enthusiastic about telling people every option they could ever have.